src/login/oidc/__mocks__/Redirector.ts

```diff
@@ -20,17 +20,11 @@
  */
 
 import { IRedirector, IRedirectorOptions } from "../Redirector";
-import INeededAction from "../../../solidSession/INeededAction";
-import INeededRedirectAction from "../../../solidSession/INeededRedirectAction";
-
-export const RedirectorResponse: INeededAction = {
-  actionType: "redirect",
-  redirectUrl: "https://coolSite.com"
-} as INeededRedirectAction;
 
 export const RedirectorMock: jest.Mocked<IRedirector> = {
   redirect: jest.fn(
-    (redirctUrl: string, redirectOptions: IRedirectorOptions) =>
-      RedirectorResponse
+    (redirctUrl: string, redirectOptions: IRedirectorOptions) => {
```

Comment:
Typo - should be 'redirectUrl'.

Reply:
Fixed in 978ac74d3e508a5dcd0a9cc51af28513b302028f